app/controllers/api/v1/conversations_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class ConversationsController < ApplicationController
+      before_action :authenticate_user!
+
+      def index
+        @conversations = current_user.sent_conversations + current_user.received_conversations
```

Comment:
Yes, it would be nice to override the `has_many`. Maybe you can try out something like this:
```
has_many :conversations, ->(user) { where(user_one_id: user.id).or(where(user_two_id: user.id)) }
```